### PR TITLE
FAQ RAT new entries

### PR DESCRIPTION
**Pull request by dsarkar**
Internal Tracking ID: [EXPOSUREAPP-6914](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6914)

### CODE REVIEW

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

File: src/data/faq_de.json [hunk 1]
```diff
@@ -204,6 +217,22 @@
...
+                        "anchor": "rat_profile_create_edit_delete",
+                        "active": true,
+                        "textblock": [
+                            "<ul><li><b>Schnelltest-Profil anlegen:</b> Um ein Schnelltest-Profil anzulegen, können Nutzer*innen auf der Startseite ihrer Corona-Warn-App unter „Test registrieren“ auf Nächste Schritte Dort finden Sie den Bereich „Schnelltest-Profil anlegen“. Dort können sie ihre persönlichen Daten, wie Name, Adresse und Telefonnummer, eintragen. Wenn alles eingetragen ist, können sie die Daten speichern. Die Corona-Warn-App erstellt daraufhin einen QR-Code aus den Daten, den Nutzer*innen an den Teststellen vorzeigen können.</li><li><b>Schnelltest-Profil entfernen:</b> Nutzer*innen können ein Schnelltest-Profil wieder entfernen, indem sie im Schnelltest-Bereich der Corona-Warn-App auf ihr Schnelltest-Profil gehen und dort 'Schnelltest-Profil entfernen' auswählen.</li><li><b>Schnelltest-Profil bearbeiten:</b> Beabsichtig ein Nutzer*innen ein Schnelltest-Profil zu bearbeiten, kann er/sie es zunächst entfernen und danach ein neues Profil anlegen.</li></ul>"
```
Comment (DerVogel2020) [inline src/data/faq_de.json:234]: The first sentence is not a sentence in German.
Beabsichtig there is a t missing.

**Review by DerVogel2020 — COMMENTED** [review]
2 small comments

File: src/data/faq.json [hunk 2]
```diff
@@ -92,6 +92,19 @@
...
+                        "active": true,
+                        "textblock": [
+                            "Rapid antigen test appointments can be booked in advance with the following partners:",
+                            "<ul><li><a href='https://meintest.brk.de' target='_blank' rel='noopener noreferrer'>Bavarian Red Cross</a></li><li><a href='https://corona-schnelltest-zentren.dm.de' target='_blank' rel='noopener noreferrer'>dm-drugstore market</a></li><li><a href='https://buergertest.ecocare.center' target='_blank' rel='noopener noreferrer'> EcoCare (Kaufland and Lidl)</a></li><li><a href='https://testbuchen.de' target='_blank' rel='noopener noreferrer'>No-Q (participating pharmacies, family doctors and other test centres)</a></li></ul>",
+                            "The following partners also participate in the rapid test integration with the Corona-Warn-App:",
+                            "<ul><li>Doctorbox</li><li>Huber Health Care</li></ul>",
+                            "The following partners also participate in the rapid test integration with the Corona Warn-App, but do not offer rapid antigen tests for the general public:",
```
Comment (DerVogel2020) [inline src/data/faq.json:104]: - is missing in Corona-Warn-App

**Review by DerVogel2020 — COMMENTED** [review]
One more comment.

**Comment by dsarkar** [comment]
image

File: src/data/faq_de.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq_de.json]: 2x thanks.

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json:104]: Guess you mean the dash - 
Thanks.

**Comment by DerVogel2020** [comment]
> image
> 

Ok.

File: src/data/faq.json (hunk 2 above)
Comment (dsarkar) [reply src/data/faq.json]: Nice suggestion! Thanks

**Comment by dsarkar** [comment]
image

**Comment by MikeMcC399** [comment]
Just checking against rc build. Stand by!

**Review by DerVogel2020 — APPROVED** [review]

File: src/data/faq.json [hunk 3]
```diff
@@ -202,6 +215,22 @@
...
+                        ]
+                    },
+                    {
+                        "title": "How can I create, edit and remove a rapid test profile?",
+                        "anchor": "rat_profile_create_edit_delete",
+                        "active": true,
+                        "textblock": [
+                            "<ul><li><b> Create a rapid test profile:</b> In order to create a rapid test profile, users can go to 'Next Steps' on the main page of their Corona-Warn-App under 'Register Test'. There you will find the area  'Create a rapid test profile'. There they can enter their personal data, such as name, address and telephone number. When everything is entered, they can save the data. The Corona-Warn-App then creates a QR code from the data that users can show at the test sites.</li><li><b>Remove rapid test profile:</b>Users can remove a quick test profile by going to their quick test profile in the quick test area of the Corona-Warn-App and selecting 'Remove rapid test profile'.</li><li><b>Edit rapid test profile:</b>If a user intends to edit a quick test profile,  he can first remove it and then create a new profile.</li></ul>"
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/faq.json:232]: Rest is your typo or typo in RC

[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/faq.json:232]: ?

File: src/data/faq.json [hunk 4]
```diff
@@ -202,6 +215,22 @@
...
+                    },
+                    {
+                        "title": "What is the purpose of a rapid test profile?",
+                        "anchor": "rat_profile",
+                        "active": true,
+                        "textblock": [
+                            "Users can create a rapid test profile with their personal data in the form of a QR code.  At test sites of participating  partners, where users do not have to make an appointment in advance,  only the QR code in the app is scanned and no further registration is required. This speeds up on-site registration. The staff at the test sites needs the personal data in order to forward it to the health authorities in case of a positive test result. For this reason, a comparison with the identity card is always necessary on site. Once created, the profile remains and can be shown again with each test."
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/faq.json:224]: thanks !

**Review by Ein-Tim — COMMENTED** [review]
I'll check against iOS Emulator in ca. 20 min.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I made some comments regarding the description of working with the Rapid Test Profile which did not fit what is shown in the Android UI. Perhaps iOS is different? This needs to be checked.

File: src/data/faq.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/faq.json]: "Rest" was my typo. Sorry. I already corrected it.

File: src/data/faq_de.json [hunk 5]
```diff
@@ -204,6 +217,22 @@
...
+                        "anchor": "rat_profile_create_edit_delete",
+                        "active": true,
+                        "textblock": [
+                            "<ul><li><b>Schnelltest-Profil anlegen:</b> Um ein Schnelltest-Profil anzulegen, können Nutzer*innen auf der Startseite ihrer Corona-Warn-App unter „Test registrieren“ auf Nächste Schritte gehen. Dort finden Sie den Bereich „Schnelltest-Profil anlegen“. Dort können sie ihre persönlichen Daten, wie Name, Adresse und Telefonnummer, eintragen. Wenn alles eingetragen ist, können sie die Daten speichern. Die Corona-Warn-App erstellt daraufhin einen QR-Code aus den Daten, den Nutzer*innen an den Teststellen vorzeigen können.</li><li><b>Schnelltest-Profil entfernen:</b> Nutzer*innen können ein Schnelltest-Profil wieder entfernen, indem sie im Schnelltest-Bereich der Corona-Warn-App auf ihr Schnelltest-Profil gehen und dort 'Schnelltest-Profil entfernen' auswählen.</li><li><b>Schnelltest-Profil bearbeiten:</b> Beabsichtigen Nutzer*innen ein Schnelltest-Profil zu bearbeiten, können sie es zunächst entfernen und danach ein neues Profil anlegen.</li></ul>"
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/faq_de.json:234]: ok

[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/faq_de.json:234]: This is how it looks on Android.  On the first screen I did check that there is no text "Schnelltest-Profil entfernen" under the "WEITER" button.

STP 1

After tapping on the three dots, then "Entfernen" appears:

STP 2

[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq_de.json:234]: Okay, seems like we have an inconsistent design between iOS and Android here.

File: src/data/faq.json [hunk 6]
```diff
@@ -202,6 +215,22 @@
...
+                        ]
+                    },
+                    {
+                        "title": "How can I create, edit and remove a rapid test profile?",
+                        "anchor": "rat_profile_create_edit_delete",
+                        "active": true,
+                        "textblock": [
+                            "<ul><li><b> Create a rapid test profile:</b> In order to create a rapid test profile, users can go to 'Next Steps' on the main page of their Corona-Warn-App under 'Register Your Test'. There you will find the area  'Create Rapid Test profile'. There they can enter their personal data, such as name, address and telephone number. When everything is entered, they can save the data. The Corona-Warn-App then creates a QR code from the data that users can show at the test sites.</li><li><b>Remove rapid test profile:</b>Users can remove a quick test profile by going to their quick test profile in the quick test area of the Corona-Warn-App and selecting 'Remove rapid test profile'.</li><li><b>Edit rapid test profile:</b>If a user intends to edit a quick test profile,  he can first remove it and then create a new profile.</li></ul>"
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/faq.json:232]: Fantastic!

File: src/data/faq.json [hunk 7]
```diff
@@ -92,6 +92,19 @@
                 "id": "rapid_antigen_test",
                 "active": true,
                 "accordion": [
+                    {
+                        "title": "Which partners participate in the rapid test integration? Which partners can I book a rapid antigen test with?",
+                        "anchor": "rat_partner",
+                        "active": true,
+                        "textblock": [
+                            "Rapid antigen test appointments can be booked in advance with the following partners:",
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/faq.json:100]: yes

File: src/data/faq_de.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (DerVogel2020) [reply src/data/faq_de.json]: Sie sollte klein sein, es geht ja um dir Nutzer*innen, die für sich ein Schnelltestprofil anlegen können.

[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq_de.json]: I think both is possible, it's not clear wether it's a "direkte Anrede" or wether it's "sie" for a group.

[the remark this answers is not in the document]
Comment (DerVogel2020) [reply src/data/faq_de.json]: If it's Direkte Anrede, it should be also changed in the other list items.

[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq_de.json]: @DerVogel2020 

This is a problem with this entry, see https://github.com/corona-warn-app/cwa-website/pull/1240#pullrequestreview-657181296. 
I suggest to open a PR afterwards fixing this.

**Comment by MikeMcC399** [comment]
I would suggest changing the style of the new FAQ entries to directly address the user. Unfortunately the last changes have now introduced inconsistencies.

In the blogs it is appropriate to describe new features applying to the third person (i.e. users), in the FAQs we should expect individual users to be looking for answers to their issues, so users should be addressed directly (i.e. you).

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
As @MikeMcC399 suggests: directly address the user.
@DerVogel2020 Sorry, this reverts your PR.

**Comment by dsarkar** [comment]
image

File: src/data/faq.json [hunk 8]
```diff
@@ -221,15 +221,15 @@
...
+                            "<ul><li><b> Create a rapid test profile:</b> In order to create a rapid test profile, you can click on 'Next Steps' on the main page of their Corona-Warn-App under 'Register Your Test'. There you will find the area 'Create Rapid Test profile'. There you can enter your personal data, such as name, address and telephone number. When everything is entered, you can save the data. The Corona-Warn-App then creates a QR code from the data that you can show at the test sites.</li><li><b>Remove rapid test profile:</b> You can remove a rapid test profile by going to your rapid test profile in the rapid test area of the Corona-Warn-App and selecting 'Remove Rapid Test Profile'.</li><li><b>Edit rapid test profile:</b> If you intends to edit a rapid test profile, you can first remove it and then create a new profile.</li></ul>"
```
Comment (DerVogel2020) [inline src/data/faq.json:232]: It should be: "If you intend".

File: src/data/faq_de.json [hunk 9]
```diff
@@ -223,16 +223,15 @@
...
+                            "Sie können ein Schnelltest-Profil mit ihren persönlichen Daten in Form eines QR-Codes anlegen. An Teststellen teilnehmender Partner, an denen Sie keinen Termin vorab vereinbaren müssen, wird dann lediglich der QR-Code in der App gescannt und es ist keine weitere Registrierung mehr erforderlich. So kann die Registrierung vor Ort beschleunigt werden. Die Mitarbeiter*innen an den Teststellen benötigen Ihre persönlichen Daten, um sie im Falle eines positiven Testergebnisses an das Gesundheitsamt weiterzuleiten. Aus diesem Grund ist vor Ort immer auch ein Abgleich mit dem Personalausweis nötig. Einmal angelegt, bleibt das Profil bestehen und kann bei jedem Test wieder vorgezeigt werden."
```
Comment (DerVogel2020) [inline src/data/faq_de.json:226]: mit **I**hren persönlichen Daten

File: src/data/faq_de.json [hunk 10]
```diff
@@ -223,16 +223,15 @@
...
-                        ]
+                            "<ul><li><b>Schnelltest-Profil anlegen:</b> Um ein Schnelltest-Profil anzulegen, können Sie auf der Startseite ihrer Corona-Warn-App unter „Test registrieren“ auf Nächste Schritte klicken. Dort finden Sie den Bereich „Schnelltest-Profil anlegen“. Dort können Sie Ihre persönlichen Daten, wie Name, Adresse und Telefonnummer, eintragen. Wenn alles eingetragen ist, können Sie die Daten speichern. Die Corona-Warn-App erstellt daraufhin einen QR-Code aus den Daten, den Sie an den Teststellen vorzeigen können.</li><li><b>Schnelltest-Profil entfernen:</b> Sie können ein Schnelltest-Profil wieder entfernen, indem Sie im Schnelltest-Bereich der Corona-Warn-App auf ihr Schnelltest-Profil gehen und dort 'Schnelltest-Profil entfernen' auswählen.</li><li><b>Schnelltest-Profil bearbeiten:</b> Beabsichtigen Sie ein Schnelltest-Profil zu bearbeiten, können Sie es zunächst entfernen und danach ein neues Profil anlegen.</li></ul>"                        ]
```
Comment (DerVogel2020) [inline src/data/faq_de.json:234]: auf **I**hr Schnelltest-Profil gehen

**Review by DerVogel2020 — COMMENTED** [review]
Ihren instead of ihren
and Ihr instead of ihr

File: src/data/faq.json (hunk 8 above)
Comment (dsarkar) [reply src/data/faq.json]: ok

File: src/data/faq_de.json (hunk 9 above)
Comment (dsarkar) [reply src/data/faq_de.json:226]: ok

File: src/data/faq_de.json (hunk 10 above)
Comment (dsarkar) [reply src/data/faq_de.json]: Thanks

File: src/data/faq_de.json [hunk 11]
```diff
@@ -204,6 +217,22 @@
...
                         ]
+                    },
+                    {
+                        "title": "Wozu dient ein Schnelltest-Profil?",
+                        "anchor": "rat_profile",
+                        "active": true,
+                        "textblock": [
+                            "Nutzer*innen können ein Schnelltest-Profil mit ihren persönlichen Daten in Form eines QR-Codes anlegen. An Teststellen teilnehmender Partner, an denen Nutzer*innen keinen Termin vorab vereinbaren müssen, wird dann lediglich der QR-Code in der App gescannt und es ist keine weitere Registrierung mehr erforderlich. So kann die Registrierung vor Ort beschleunigt werden. Die Mitarbeiter*innen an den Teststellen benötigen die persönlichen Daten, um sie im Falle eines positiven Testergebnisses an das Gesundheitsamt weiterzuleiten. Aus diesem Grund ist vor Ort immer auch ein Abgleich mit dem Personalausweis nötig. Einmal angelegt, bleibt das Profil bestehen und kann bei jedem Test wieder vorgezeigt werden."
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/faq_de.json]: Thanks, will adopt later maybe.

**Comment by dsarkar** [comment]
image

**Pull request merged**